pkg/settings/settings.go

```diff
@@ -324,3 +326,63 @@ func GetCSIDriverInfo(provisioner string) (*CSIDriverInfo, error) {
 	}
 	return csiDriverInfo, nil
 }
+
+type StrategyType string
+
+const (
+	// Do no preloading
+	SkipType StrategyType = "skip"
+
+	// Preloading one node at a time
+	SequentialType StrategyType = "sequential"
+
+	// Preloading multiple nodes starts at the same time
+	ParallelType StrategyType = "parallel"
+)
+
+type PreloadStrategy struct {
+	Type StrategyType `json:"type,omitempty"`
+
+	// Concurrency only takes effect when ParallelType is specified. Default to
+	// 0, which means "full scale." Any value larger than the number of the cluster
+	// nodes or smaller than 0 will be treated as 0.
```

Comment:
We have validator, why not just throw error if value < 0 ?

Reply:
Looks like you already make concurrently < 0 invalid in validator, we should modify the comment here.

Reply:
Good point. Let me align the implementation between the validator and the controller (also the comment).